src/styles/main.scss

```
@@ -1,3 +1,76 @@
 body {
   margin: 0;
 }
+
+@for $i from 1 through 31 {
+  .calendar__date-value:nth-child(#{$i}):before {
+    content: "#{$i}";
+    font-size: 30px;
+    font-family: "Arial", sans-serif;
+  }
+}
+
+$days: (
+  mon: "Monday",
+  tue: "Tuersday",
+  wed: "Wensday",
+  thu: "Thursday",
+  fri: "Friday",
+  sat: "Saterday",
+  sun: "Sunday"
+);
+
+.value {
+  width: 100px;
+  margin-right: 10px;
+}
+
+@each $name, $day in $days {
+  .calendar__day-#{$name}::before {
+    content: "#{$day}";
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 20px;
+    color: red;
+  }
+}
+
+.calendar__day {
+  display: flex;
+}
+
+@for $i from 28 through 31 {
+  .month-length:nth-child(#{$i}):before {
+    content: " ";
+  }
+}
+
+.calendar__date {
+  display: grid;
+  grid-template-columns: repeat(7, 0fr);
+}
+
+.calendar__date-value {
+  background-color: #eee;
```

create variables for colors
```suggestion
  background-color: $c-someColor;
```